docs/EmbeddedSwift/EmbeddedSwiftStatus.md

```diff
@@ -10,8 +10,8 @@ For an introduction and motivation into Embedded Swift, please see "[A Vision fo
 
 This status table describes which of the following standard library features can be used in Embedded Swift:
 
-| **Swift Standard Library Feature**          | **Currently Supported In Embedded Swift** |
-|---------------------------------------------|-----------------------------------------------------|
+| **Swift Standard Library Feature**                         | **Currently Supported In Embedded Swift?**          |
```

Comment:
Fun fact, you don't need to pad the columns out. Markdown will render this nicely:

```md
| **Foo** | **Bar** |
|-|-|
| Alpha. | Beta. |
| Gamma delta epsilon eridani. | Live long and prosper. |
```

| **Foo** | **Bar** |
|-|-|
| Alpha. | Beta. |
| Gamma delta epsilon eridani. | Live long and prosper. |